addons/nametags/functions/fnc_getCachedFlags.sqf

```diff
@@ -0,0 +1,64 @@
+/*
+ * Author: <N/A>
```

Comment:
https://github.com/acemod/ACE3/blob/master/addons/nametags/functions/fnc_onDraw3d.sqf#L2 That guy